pkg/domain/entities/images.go

```diff
@@ -242,8 +242,9 @@ type ImageSearchReport = entitiesTypes.ImageSearchReport
 
 // Image List Options
 type ImageListOptions struct {
-	All    bool     `json:"all" schema:"all"`
-	Filter []string `json:"Filter,omitempty"`
+	All                bool     `json:"all" schema:"all"`
+	ExtendedAttributes bool     // no json needed
+	Filter             []string `json:"Filter,omitempty"`
```

Comment:
Can remove all the json and schema tags here, this is super confusion because non of this is ever serialized and send over the API, we have proper bindings type that is used to set the options in the API call.

Reply:
i didnt add 2/3 of those ... i only added the one line.

Reply:
yes but now I saw the error and since you touch this anyway you can fix this for all fields, saying no json needed for one field when no fields needs it is confusing!